src/main/java/com/nautilus/model/User.java

```diff
@@ -0,0 +1,16 @@
+package com.nautilus.model;
+
+import lombok.Data;
+
+import javax.persistence.*;
```

Comment:
?

Reply:
idea make it when i push ```cmd+option+L```

Reply:
Did you change max values of imports in settings of the idea?